app/kubemci/cmd/create.go

```diff
@@ -149,6 +149,14 @@ func runCreate(options *CreateOptions, args []string) error {
 		return err
 	}
 
+	if options.Validate {
+		if newEnough, err := validations.ServerVersionsNewEnough(clients); err != nil {
+			return fmt.Errorf("cluster version could not be validated: %s", err)
+		} else if !newEnough {
+			return fmt.Errorf("not all cluster versions new enough for kubemci")
```

Comment:
We should be explicit about what "not new enough" means. Say they should be > 1.8.1 and not 1.10.0?

Reply:
I wanted to keep the knowledge of the exact supported versions in a minimum of locations. I've centralized it all in validations.go. That prints
fmt.Println("Cluster", key, "is not running a supported kubernetes version. Need at least 1.8.1.")
I'll the testing and logging about 1.10.0.

Reply:
hmm. Seems like you can just remove the bool here. If err !=nil, then return that as is.
ServerVersionsNewEnough already returns meaningful error messages.